blueprints/ember-frost-lts/index.js

```diff
@@ -1,4 +1,5 @@
 'use strict'
```

Comment:
You can avoid the use of a `self` variable by using fat arrow functions https://strongloop.com/strongblog/an-introduction-to-javascript-es6-arrow-functions/ (see the Lexical this section)